package.json

```diff
@@ -36,9 +37,11 @@
     "@commitlint/config-conventional": "^8.1.0",
     "@types/jest": "^24.0.12",
     "codecov": "^3.4.0",
+    "cucumber": "0.5.x",
     "generate-changelog": "^1.7.1",
     "husky": "^3.0.0",
     "jest": "^24.7.1",
+    "jmespath": "^0.15.0",
```

Comment:
Nit: is `jmespath` used anywhere?

Reply:
The package `jmespath` is used for verifying `it should have attribute`
* feature: https://github.com/aws/aws-sdk-js-v3/blob/0ade538f67f817ae3b683b206ddfc6985bc7e6d7/features/dynamodb/tables.feature#L37
* step definition: https://github.com/aws/aws-sdk-js-v3/blob/0ade538f67f817ae3b683b206ddfc6985bc7e6d7/features/dynamodb/step_definitions/dynamodb.js#L77